.circleci/config.yml

```diff
@@ -139,6 +139,17 @@ jobs:
             yarn ganache > /dev/null &
             cd source/wrapper && yarn test
 
+  debugger_tests:
+    <<: *container_config
+    steps:
+      - attach_workspace:
+          at: *working_directory
+      - run:
+          name: Run Tests
+          command: |
+            yarn ganache > /dev/null &
+            sleep 10s && cd utils/debugger && yarn test
```

Comment:
you can remove the sleep in order_util_tests now that it's not relying on ganache anymore.